source/Firmata/UwpFirmata.cpp

```diff
@@ -340,6 +534,20 @@ UwpFirmata::write(
 //******************************************************************************
 
 
+String ^
+UwpFirmata::createStringFromMbs(
+    uint8_t *mbs,
+    size_t len
```

Comment:
Else where you use a trailing underscore to indicate function parameters.  Can you include them here to be consistent?